decisions/Gondor.txt

```diff
@@ -1691,6 +1691,11 @@ political_decisions = {
 		allow = {
 			war = no
 			expansion_2 = 1
+			HRD = {
+				all_core = {
+					empty = no
+				}
+			}
```

Comment:
I'd prefer that we do an empty check in the potential. With how it's written here, it won't actually change when the decision can be taken

Reply:
It does change it though

Reply:
Discarded.